pkgs/top-level/aliases.nix

```diff
@@ -930,6 +930,7 @@ mapAliases ({
   telnet = inetutils; # added 2018-05-15
   terminus = throw "terminus has been removed, it was unmaintained in nixpkgs"; # added 2021-08-21
   terraform_1_0_0 = terraform_1_0; # added 2021-06-15
+  terraform_1_0 = terraform_1; # added 2021-12-08
```

Comment:
This seems somewhat unintuitive for me now that terraform_1 is actually 1.1 rather than 1.0.

Reply:
would you prefer `terraform_1_x`?

Reply:
I mean, the fact that using `terraform_1_0` now installs a Terraform 1.1 is a problem IMHO. `terraform_1` is perfectly fine though.

Reply:
It's the same issue with `terraform_1_0_0`. Let's `throw "renamed to terraform_1"` instead for both.

Reply:
Sorry for the delay, I've updated the aliases PTAL